### PR TITLE
Update english-chinese-comparison-table.md

### PR DESCRIPTION
**Pull request by xiaoya-Esther**
Signed-off-by: Xiaoya <xiaoyaxiaesther@gmail.com>

### CODE REVIEW

File: chinese/english-chinese-comparison-table.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 | English | 中文 |
 | --- | --- |
+| metric | 指标 |
+| visualizations | 可视化效果 |
+| tools providing the metric | 提供指标的工具 |
+| **Evolution**
```
Comment (eyehwan) [inline chinese/english-chinese-comparison-table.md:10]: **Evolution**

Why do we need to add it?

File: chinese/english-chinese-comparison-table.md [hunk 2]
```diff
@@ -14,6 +18,9 @@
 | references | 参考资料 |
 | repository | 代码仓 |
 | review | 审查，审核 |
-| tools providing the metric | 提供度量的工具 |
-| visualizations | 可视化效果 |
 | committer | committer |
+| **Diversity and Inclusion**
```
Comment (eyehwan) [inline chinese/english-chinese-comparison-table.md:22]: **Diversity and Inclusion**

Not all the words are from the same working group, so I prefer not classify like this. 
What do you think? Xiaoya.

Comment (xiaoya-Esther) [reply chinese/english-chinese-comparison-table.md:22]: My concern is one word in different working groups may refer to different meanings, e.g. Issue doesn't need to translate in Evolution Working Group, but may refer to actual problems in other working groups.

Comment (xiaoya-Esther) [reply chinese/english-chinese-comparison-table.md:22]: We can further discuss this. I‘m still not sure how we can maintain this table in a better way:)

Comment (eyehwan) [reply chinese/english-chinese-comparison-table.md:22]: > My concern is one word in different working groups may refer to different meanings, e.g. Issue doesn't need to translate in Evolution Working Group, but may refer to actual problems in other working groups.

I take reference from here:
https://docs.github.com/cn/rest/reference/issues
Issue -> 议题。

**Pull request closed without merging**